### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by hamoid**
I'll open a new PR weekly and collect guide changes there. Maybe to be merged on weekends?

### CODE REVIEW

File: src/main/kotlin/docs/11_Advanced_Topics/C06_Compute_shaders.kt [hunk 1]
```diff
@@ -15,54 +15,81 @@ shader support only works on systems that support OpenGL 4.3 or higher. This exc
...
-    @Text """Here we provide a quick break-down on compute shader use. This text is preliminary and will be improved
-shortly."""
+    @Text """
+This example is composed of two code blocks. The first block is a compute 
+shader program written in GLSL which produces an `outputImg` by mixing three input 
+colors:
+* A pixel sampled from `inputImg`.
+* A `fillColor` sent as a uniform from Kotlin.
+* A color generated by calculating the cosine of the coordinates of the pixel 
+curretly being processed.
```
Comment (Vechro) [inline src/main/kotlin/docs/11_Advanced_Topics/C06_Compute_shaders.kt:25]: curretly -> currently

File: src/main/kotlin/docs/11_Advanced_Topics/C06_Compute_shaders.kt [hunk 2]
```diff
@@ -15,54 +15,81 @@ shader support only works on systems that support OpenGL 4.3 or higher. This exc
...
+The second code block is an OPENRNDR program making use of the compute shader:
+1. It creates a compute shader program from a file. 
+1. Initializes the input buffer using a image loaded from disk.
+1. Inside the `extend` block (called multiple times per second) some uniforms 
+are updated: a `fillColor`, the time in 
+`seconds`, the `inputImg` and the `outputImg`.  In this example only 
+the time changes on every frame.
+1. The compute shader is executed, writing the result to the 
+`outputBuffer`
+1. Finally, the result is displayed by calling `drawer.image()`
```
Comment (Vechro) [inline src/main/kotlin/docs/11_Advanced_Topics/C06_Compute_shaders.kt:65]: Do you want to punctuate the items in the list or not? However you answer, make sure you keep it consistent across lists. The last 2 items stand out here for not having full stops.

File: src/main/kotlin/docs/04_Drawing_basics/C08_DrawingPrimitivesBatched.kt [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+    """
+
+    @Text """
+    ## Batched circles
+
+    This example makes use of `Circle` (a class with properties like 
+    `center` and `radius` and some useful methods), not to be confused with 
+    `drawer.circle()` (a method that draws pixels on the screen). 
+     It is possible to construct a `Circle` using `center` and `radius` but 
+     also with two or three `Vector2` points.
```
Comment (Vechro) [inline src/main/kotlin/docs/04_Drawing_basics/C08_DrawingPrimitivesBatched.kt:40]: I feel this sentence doesn't really tell the user what difference between the two constructors is. How about...

> It is possible to construct a `Circle` by providing a `center` position and a `radius` but 
     also with two or three `Vector2` points that are used for deriving the circumference of a `Circle`.

Comment (hamoid) [reply src/main/kotlin/docs/04_Drawing_basics/C08_DrawingPrimitivesBatched.kt:40]: Much better, thanks!

File: src/main/kotlin/docs/11_Advanced_Topics/C06_Compute_shaders.kt [hunk 4]
```diff
@@ -15,54 +15,81 @@ shader support only works on systems that support OpenGL 4.3 or higher. This exc
...
     media/compute-shaders-001.png
     """
 
+    @Text """
+The second code block is an OPENRNDR program making use of the compute shader:
+1. It creates a compute shader program from a file. 
+1. Initializes the input buffer using a image loaded from disk.
+1. Inside the `extend` block (called multiple times per second) some uniforms 
+are updated: a `fillColor`, the time in 
+`seconds`, the `inputImg` and the `outputImg`.  In this example only
```
Comment (Vechro) [inline src/main/kotlin/docs/11_Advanced_Topics/C06_Compute_shaders.kt:61]: Two spaces between the sentences? Perhaps a minuscule problem but it does bother me when I'm reading something lol.

Comment (hamoid) [reply src/main/kotlin/docs/11_Advanced_Topics/C06_Compute_shaders.kt:61]: They should be collapsed by html, but I'll get rid of it :-)

**Pull request merged**